mne/epochs.py

```diff
@@ -1276,8 +1276,7 @@ def __getitem__(self, key):
                                        'event matches all criteria.')
             select = np.any(np.atleast_2d([epochs._key_match(k)
                                            for k in key]), axis=0)
-            epochs.name = ('+'.join(key) if epochs.name == 'Unknown'
-                           else 'epochs_%s' % '+'.join(key))
+            epochs.name = '+'.join(key)
```

Comment:
the line above is `if isinstance(key, (list, tuple)) and isinstance(key[0], string_types):` so it should be checked. in any case this PR doesn't affect that bit, since all it really does is change a conditional...